L1Trigger/L1THGCalUtilities/python/hgcalTriggerNtuples_cfi.py

```diff
@@ -92,8 +92,9 @@
     NtupleName = cms.string('HGCalTriggerNtupleHGCMulticlusters'),
     Multiclusters = cms.InputTag('hgcalBackEndLayer2Producer:HGCalBackendLayer2Processor3DClustering'),
     EGIdentification = egamma_identification_histomax.clone(),
-    FillLayerInfo = cms.bool(False),
-    FillInterpretationInfo = cms.bool(True)
+    FillLayerInfo = cms.bool(True),
```

Comment:
`FillLayerInfo` should be kept as `False`, at least in this PR (since this change is not related to the emulator)

Reply:
Apologies, this slipped through to a commit by accident, fixed.

Reply:
Seems that it is still set to `True`